### ISSUE TITLE
Add clarity or conformance level on accessibility A, AA, AAA for the for TV Apps 

### ISSUE THREAD

**Issue by Neha** [w3c/wcag#4065]
WCAG guidelines are clear for the website, and mobile apps. However, with the increase use of the OTT platform, the accessibility levels and guidelines are missing for TV apps. Eg:

1. **Focused ring**: what should be the color contrast for the TV apps for A, AA, AAA
2.  **Color contrast:** for background color and text
3.  **Font-size:** Minimum font-size 
4. **Navigation**: How the navigation should be 
5. **Hints**: How the hints should be, and position
6. **Screenreaders**: Usage of screenreaders

**Comment by patrickhlauke** [comment]
WCAG is technology-agnostic in its principles and criteria. Your points 1 and 2 are already covered with existing criteria. Pointer 3, 4, 5, 6 go outside of the scope of what WCAG in general mandates.

**Comment by GreggVan** [comment]
Correct. WCAG has always driven to be as agnostic as possible to the technologies used on the web.

However, this should not be confused with being **technology-comprehensive.**



What I mean by that is that:
- WCAG assumes things that are not always true when not talking about web content
- For example - it assumes there is a user agent.
     - so we eliminated 4.1.1 because we are assuming there is a browser that repairs all those things. 
     - we assume there is always a keyboard or keyboard equivalent
     - we assume that there are AT that can be installed on the devices that house the browser (or that the user can choose a device that has the AT they need — and use that browser to access the content

- If you try to apply WCAG to non-web content and non-web software and mobile apps and to ICT in general
    - many of the provisions apply directly
           - others apply with simple words substitutions 
           - still others apply awkwardly or do not quite work
    - but most importantly  - there are all sorts of other things that WCAG does not even mention
                    - all aspects of hardware
                    - all aspects dealing with “closed functionality” products (hardware and software) that do not accept AT
           - how to provide access when there isn’t a browser or AT  (users can choose which hardward to access the web mostly but cannot chose a different technology when they confront a kiosk or copy machine or thermostat or television or any other ICT. 

WCAG2ICT is written specifically to raise these issues — but it only points them out and does not solve them or make WCAG comprehensive

So it is great for web content

it is (very) useful for non-web docs and software **_and mobile apps_** — but has to be adapted and **_does not cover all the areas / aspects._** 

best
